### ISSUE TITLE
Part of marker remains visible on Precipitation Nowcasting

### ISSUE THREAD

**Issue by 2013ZRZ** [breezy-weather/breezy-weather#1001]
### Steps to reproduce

1.See Precipitation Nowcasting
2.Click on the chart

### Expected behavior

Nothing is left when you loosen it.

### Actual behavior

There's a line.

### Weather source used

AccuWeather

### Breezy Weather version

5.2.1 armeabi_v7a

### Android version

EMUI 5.1.3 Android Nougat

### Device

Honor 6a

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue.
- [X] I checked that my issue is not covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document.
- [X] I have updated the app to **[version 5.2.1 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)** and I can reproduce the issue in this version.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance), unless my issue only happens with a specific language. I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by 2013ZRZ** [comment]
![IMG_20240422_201236](https://github.com/breezy-weather/breezy-weather/assets/130883435/b36fe588-1b56-4b0a-8c0e-b11ce0ffd902)

**Comment by 2013ZRZ** [comment]
I'm sorry. The version information in that comment is incorrect. I'm inputted incorrectly.
Now I'm changed it.

**Comment by papjul** [comment]
Alright, can you see a marker when you hold tap on the chart?

**Comment by 2013ZRZ** [comment]
Yes. When the bubble passes by, a light line will appear. When the lines overlap, the colors of the lines become darker (see Screenshot).

**Comment by papjul** [comment]
Could you download the sample APK from the chart library we use here: https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.16

And let us know if you can reproduce the same issue in any of the examples with a marker available?

**Comment by 2013ZRZ** [comment]
No, I can't.
But I didn't use dark mode, so it's possible that I can't see the marker because the color left is the same as the background color.

**Comment by papjul** [comment]
> No, I can't.

You answer to "Can you reproduce", right?

>  I can't see the marker 

So, you can see a marker in Breezy Weather but not in any of the examples of the sample app?

**Comment by 2013ZRZ** [comment]
> > No, I can't.
> 
> You answer to "Can you reproduce", right?
> 
> > I can't see the marker
> 
> So, you can see a marker in Breezy Weather but not in any of the examples of the sample app?

Yes. 

Has chart of vico what breezy weather used a bubble? My Android version didn't support dark mode. For the reasons I mentioned earlier, I can't see the bubble.

**Comment by papjul** [comment]
What does the marker have to do with the dark mode?

**Comment by 2013ZRZ** [comment]
In vico, I can't see the bubble.
There are two possibilities: 
1. There is no bubble.
2. If I use the dark mode, the background color isn't white, so I can see the bubble. But in light mode, the background color is white. That I can't see the bubble. However, the possible premise is that the color of the bubble is white and will not change with the theme.

I may not have made it clear in the previous comments. I mean that there are bubbles in the chart data in Breezy Weather, but there are no bubbles in similar charts in the vico sample application (maybe because I can't see them, as mentioned in the previous paragraph).

**Comment by 2013ZRZ** [comment]
> In vico, I can't see the bubble. There are two possibilities:
> 
> 1. There is no bubble.
> 2. If I use the dark mode, the background color isn't white, so I can see the bubble. But in light mode, the background color is white. That I can't see the bubble. However, the possible premise is that the color of the bubble is white and will not change with the theme.
> 
> I may not have made it clear in the previous comments. I mean that there are bubbles in the chart data in Breezy Weather, but there are no bubbles in similar charts in the vico sample application (maybe because I can't see them, as mentioned in the previous paragraph).

Because the line of Precipitation Nowcasting in Breezy Weather is the same color as the bubble, I think there is such a line in vico. I just can't see it because the color is close to or the same as the background color.

**Comment by papjul** [comment]
Could you take a screenshot of:
- The marker in Breezy Weather
- The marker in "Chart 2" (or whatever is shown when you tap on the chart)

Thanks!

**Comment by papjul** [comment]
Additional question: can you reproduce in v5.2.3 of Breezy Weatger? I just installed it on an Android 5.0 phone, which is an even older version than yours, and I don't have this issue, with or without dark mode

**Comment by papjul** [comment]
I notice on your second video that you don't have any background in the marker, just the text. That's a different issue, but this time, this is something I can reproduce on my 13-years old phone.

I’m sorry, but I’m out of ideas. I don’t know if that’s an EMUI, a faulty hardware driver, or something wrongly handled on the lib side. I will leave this open to other contributors if they have an idea, but it’s mostly a cosmetic issue so it should not prevent the general usage of the app.

**Comment by papjul** [comment]
Still happening in v5.2.6?

**Comment by 2013ZRZ** [comment]
> Still happening in v5.2.6?

Yes.

I just tested it on three other devices, and one of them can reproduce this problem. I found that the screen aspect ratio of two mobile phones that can reproduce the problem is **16:9**, while the normal screen aspect ratios of the two mobile phones are **13:6** and **21:9**. So I think this problem may be related to the aspect ratio of the screen.

**Comment by papjul** [comment]
16:9 is a common ratio; can't reproduce on my 16:9.

What are those two devices so I can check if I see anything else in common?

**Comment by 2013ZRZ** [comment]
> What are those two devices so I can check if I see anything else in common?

The following are two devices that can reproduce this problem and some of their parameters:

HUAWEI nova youth
* CPU: HiSilicon Kirin 658
* OS: Android 8.0.0
* Screen: 5.2 inch
* Resolution Ratio: 1920*1080

Honor 6A
* CPU: Qualcomm Snapdragon 430
* OS: Android 7.0
* Screen: 5 inch
* Resolution Ratio: 1280*720
